src/boilerplates/create_wallet_linking_embedded_wallet_react.ts

```diff
@@ -0,0 +1,56 @@
+
+import { Command } from "commander";
+import { promptForGoogleClientIdWithLogs, promptForProjectAccessKeyWithLogs, promptForWaaSConfigKeyWithLogs, promptForWalletConnectIdWithLogs, promptUserKeyCustomizationDecision, writeDefaultKeysToEnvFileIfMissing, writeToEnvFile } from "../utils";
+import { EnvKeys } from "../utils/types";
+
+import shell from "shelljs";
+
+const WALLET_LINKING_EMBEDDED_WALLET_REACT_REPO_URL = "https://github.com/0xsequence-demos/demo-embedded-wallet-linking";
+
+export async function createWalletLinkingEmbeddedWalletReact(program: Command, options: any) {
+    let waasConfigKey = options.waasConfigKey;
+    let projectAccessKey = options.projectAccessKey;
+    let googleClientId = options.googleClientId;
+    let walletConnectId = options.walletConnectId;
+
+    const userWantsToConfigureTheirKeys = await promptUserKeyCustomizationDecision();
+    
+    if (userWantsToConfigureTheirKeys) {
+        waasConfigKey = await promptForWaaSConfigKeyWithLogs(waasConfigKey);
+        projectAccessKey = await promptForProjectAccessKeyWithLogs(projectAccessKey);
+        googleClientId = await promptForGoogleClientIdWithLogs(googleClientId);
+        walletConnectId = await promptForWalletConnectIdWithLogs(walletConnectId);
+    }
+
+    console.log("Cloning the repo to `demo-embedded-wallet-linking`...");
+
+    shell.exec(`git clone ${WALLET_LINKING_EMBEDDED_WALLET_REACT_REPO_URL} demo-embedded-wallet-linking`, { silent: !options.verbose });
+    
+    shell.cd("demo-embedded-wallet-linking");
+    
+    shell.exec(`touch .env`, { silent: !options.verbose });
+
+    console.log("Configuring your project...");
+
+    const envExampleContent = shell.cat('.env.example').toString();
+    const envExampleLines = envExampleContent.split('\n');
+
+    const envKeys: EnvKeys = {
+        "VITE_WAAS_CONFIG_KEY": waasConfigKey || undefined,
+        "VITE_PROJECT_ACCESS_KEY": projectAccessKey || undefined,
+        "VITE_GOOGLE_CLIENT_ID": googleClientId || undefined,
+        "VITE_WALLET_CONNECT_ID": walletConnectId || undefined,
+    };
```

Comment:
This envKeys is wrong. This is the correct envKeys:

const envKeys: EnvKeys = {
        "VITE_WAAS_CONFIG_KEY": waasConfigKey || undefined,
        "VITE_PROJECT_ACCESS_KEY": projectAccessKey || undefined,
        "VITE_GOOGLE_CLIENT_ID": googleClientId || undefined,
        "VITE_WALLET_CONNECT_ID": walletConnectId || undefined,
    };
    
In the repo, in the .env.example, VITE_WALLET_CONNECT_ID doesn’t exist. There, it’s named VITE_WALLET_CONNECT_PROJECT_ID. I would probably change that environment variable in the repository itself since it would be good for the same environment variable to have the same name across all repositories.